### ISSUE TITLE
special symbols in text

### ISSUE THREAD

**Issue by 152334H** [152334H/DL-Art-School#10]
Notable problem of the base tortoise model: it does not handle `*emphasis*` or special symbols like `+` or `=` or `&` elegantly. Lines like:
* `8 + 5 = 13`
* `him & his wife`
* `I *really* do not like this`

all perform oddly poorly.

I am not sure if fine-tuning could fix this. Personally, I think you would need to at least fine-tune both the CLVP && GPT model to fix this.

**Comment by xenotropic** [comment]
This seems like it would be easier to fix with text preprocessing / normalization (for plus, equals, ampersand) and bold emphasis with bracket prompt engineering, "I [emphatically] do not like this".

**Comment by 152334H** [comment]
yeah my existing hobby projects all do that kind of thing, but there's no inherent reason why transformers shouldn't be able to just handle the symbols
